docs/decision_environment.rst

```diff
@@ -0,0 +1,92 @@
+
+Lets run a rulebook using this decision environment, lets make sure we have a local inventory that has localhost in it we're going to use:
+
+.. code-block:: shell
+
+    echo "localhost ansible_connection=local" > inventory
+
+Now lets run a rulebook using this decision environment:
+
+.. code-block:: shell
+
+    docker run -it --rm -v ./inventory:/tmp/inventory ansible-execution-env:latest ansible-rulebook -r ansible.eda.hello_events -i /tmp/inventory
+
+
+Using your own rulebooks and projects with the decision environment
+-------------------------------------------------------------------
+
+The minimal decision environment is a good starting point, but you will likely want to add your own rulebooks and projects to it.
+
+.. note::
+
+    Have a look at the `Ansible Builder Execution Environment Definition <https://ansible-builder.readthedocs.io/en/latest/definition/>`_ for details on how to add collections and dependencies to your decision environment.
+
+.. code-block:: yaml
+
+    ---
+    version: 3
+
+    images:
+    base_image:
+        name: 'minimal-decision-environment:latest'
+    dependencies:
+        python:
+            - pywinrm
+        system:
+            - iputils [platform:rpm]
+        galaxy:
+            collections:
+                - name: my_namespace.my_awesome_collection
+                - name: community.windows
+                - name: ansible.utils
+                version: 2.10.1
+    options:
+        container_init:
+            entrypoint: '["/opt/builder/bin/entrypoint","ansible-rulebook", "-r", "my_namespace.my_awesome_collection.my_rulebook", "-i", "/tmp/inventory"]'
+
+This shows an example where you may have your own Collection that contains rulebooks and playbooks but need to bring them together with some other collections
+and some python and system dependencies.
+
+You could also use Builder to add your own rulebooks and playbooks to the decision environment via `addtional-build-steps<https://ansible-builder.readthedocs.io/en/latest/definition/#additional-build-steps>`_
+and then making use of Containerfile commands to ADD or COPY to get the files into the environment.
+
+.. code-block:: yaml
+
+    ---
+    version: 3
+
+    images:
+    base_image:
+        name: 'minimal-decision-environment:latest'
+    dependencies:
+        python:
+            - pywinrm
+        system:
+            - iputils [platform:rpm]
+        galaxy:
+            collections:
+                - name: community.windows
+                - name: ansible.utils
+                version: 2.10.1
+    additional_build_steps:
+        prepend_builder:
+            - 'RUN mkdir -p /opt/ansible/my_rulebooks'
+            - 'COPY my_rulebook.yml /opt/ansible/my_rulebooks'
+    options:
+        container_init:
+            entrypoint: '["/opt/builder/bin/entrypoint","ansible-rulebook", "-r", "/opt/ansible/my_rulebooks/my_rulebook.yml", "-i", "/tmp/inventory"]'
```

Comment:
I understood that you were going to omit the entrypoint.

Reply:
I did remove it from the minimal container definition. I left it here as an example of how you might add this thing if you wanted it to behave a different way. See the note right below this.

Reply:
I'll remove it from this one